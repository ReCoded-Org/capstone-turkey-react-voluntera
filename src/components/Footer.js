import { Link } from 'react-router-dom';
import logo from '../assets/logov.png';
import Linkedin, { Facebook, Github } from './Icons';

function Footer() {
  return (
    <div className="h-80 bg-purple-600 flex">
      <div className="container mx-auto px-2 sm:px-6 lg:px-8 flex gap-5 items-center justify-around">
        <div className="flex flex-col gap-4">
          {/* logo section */}
          <div className="flex-shrink-0 flex gap-2 items-center justify-around bg-purple-900 px-3 py-2 rounded-md w-44">
            <img
              className="block h-8 w-auto rounded-md"
              src={logo}
              alt="Workflow"
            />
            <span className="text-2xl text-white font-semibold">Valuntera</span>
          </div>
          <p className="text-white">
            Copyright Â© Valuntera All rights reserved{' '}
          </p>
          <div className="flex gap-2">
            <Linkedin />
            <Facebook />
            <Github />
          </div>
        </div>
        {/* menu section */}
        <div className="flex flex-col gap-1 justify-start">
          <h1 className="text-white text-xl mb-4">MENU</h1>
          <Link
            to="/"
            className="text-white font-semibold hover:text-purple-900"
          >
            Home
          </Link>
          <Link
            to="/about"
            className="text-white font-semibold hover:text-purple-900"
          >
            About
          </Link>
          <Link
            to="/contact"
            className="text-white font-semibold hover:text-purple-900"
          >
            Contact
          </Link>
        </div>
        {/* supports */}
        <div className="flex flex-col gap-1 justify-start">
          <h1 className="text-white text-xl mb-4">SUPPORT</h1>
          <Link
            to="support"
            className="text-white font-semibold hover:text-purple-900"
          >
            Support Center
          </Link>
          <Link
            to="/help"
            className="text-white font-semibold hover:text-purple-900"
          >
            Help Desk
          </Link>
          <Link
            to="/faqs"
            className="text-white font-semibold hover:text-purple-900"
          >
            FAQs
          </Link>
        </div>
      </div>
    </div>
  );
}

export default Footer;
