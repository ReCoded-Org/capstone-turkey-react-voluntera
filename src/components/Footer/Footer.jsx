import { Link } from 'react-router-dom';

import logo from '../../assets/images/logov.png';
import {
  ABOUT_ROUTE,
  CONTACT_ROUTE,
  FAQS_ROUTE,
  HOME_ROUTE,
} from '../../routes';
import Linkedin, { Github } from '../Icons';

function Footer() {
  return (
    <div className=" bg-purple-600 flex py-10 px-10">
      <div className="container mx-auto px-2 sm:px-8  lg:px-8 flex flex-col gap-5 items-center justify-between sm:flex-col md:flex-row sm:flex ">
        <div className="flex flex-col gap-4">
          {/* logo section */}
          <Link to={HOME_ROUTE}>
            <div className="flex-shrink-0 flex gap-2 items-center justify-around bg-purple-900 px-3 py-2 rounded-md w-44">
              <img
                className="block h-8 w-auto rounded-md"
                src={logo}
                alt="logo"
              />
              <span className="text-2xl text-white font-semibold">
                Valuntera
              </span>
            </div>
          </Link>
          <p className="text-white">
            Copyright Â© Valuntera All rights reserved{' '}
          </p>
          <div className="flex gap-2">
            <a
              href="https://www.linkedin.com/school/re-coded/"
              rel="noreferrer"
              target="_blank"
            >
              <img
                src="https://avatars.githubusercontent.com/u/35921230?s=200&v=4"
                alt="recoded"
                className="h-10 w-auto rounded-full"
              />
            </a>
            <a
              href="https://www.linkedin.com/school/re-coded/"
              rel="noreferrer"
              target="_blank"
            >
              <Linkedin />
            </a>

            <a
              href="https://github.com/ReCoded-Org/capstone-turkey-react-voluntera"
              target="_blank"
              rel="noreferrer"
            >
              {' '}
              <Github />
            </a>
          </div>
        </div>
        {/* menu section */}
        <div className="flex flex-col w-80 px-5 gap-1 sm:w-80 sm:px-5 ">
          <h1 className="text-white text-xl mb-4">MENU</h1>
          <Link
            to={HOME_ROUTE}
            className="text-white font-semibold hover:text-purple-900"
          >
            Home
          </Link>
          <Link
            to={ABOUT_ROUTE}
            className="text-white font-semibold hover:text-purple-900"
          >
            About
          </Link>
          <Link
            to={CONTACT_ROUTE}
            className="text-white font-semibold hover:text-purple-900"
          >
            Contact
          </Link>
        </div>
        {/* supports */}
        <div className="flex flex-col w-80 px-5 gap-1 justify-start sm:w-80 sm:px-5">
          <h1 className="text-white text-xl mb-4">SUPPORT</h1>
          <Link
            to={FAQS_ROUTE}
            className="text-white font-semibold hover:text-purple-900"
          >
            FAQs
          </Link>
          <Link
            to="support"
            className="text-white font-semibold hover:text-purple-900"
          >
            Support Center
          </Link>
          <Link
            to="/help"
            className="text-white font-semibold hover:text-purple-900"
          >
            Help Desk
          </Link>
        </div>
      </div>
    </div>
  );
}

export default Footer;
