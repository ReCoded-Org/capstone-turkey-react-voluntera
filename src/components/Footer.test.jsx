import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import Footer from './Footer';

test('should render the footer', () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Footer />
      </BrowserRouter>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
