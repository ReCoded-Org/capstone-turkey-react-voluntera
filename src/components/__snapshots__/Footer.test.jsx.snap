// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the footer 1`] = `
<div
  className=" bg-purple-600 flex py-10 px-10"
>
  <div
    className="container mx-auto px-2 sm:px-8  lg:px-2 flex flex-col gap-5 items-center justify-between sm:flex-col md:flex-row sm:flex "
  >
    <div
      className="flex flex-col gap-4"
    >
      <div
        className="flex-shrink-0 flex gap-2 items-center justify-around bg-purple-900 px-3 py-2 rounded-md w-44"
      >
        <img
          alt="Workflow"
          className="block h-8 w-auto rounded-md"
          src="logov.png"
        />
        <span
          className="text-2xl text-white font-semibold"
        >
          Valuntera
        </span>
      </div>
      <p
        className="text-white"
      >
        Copyright Â© Valuntera All rights reserved
         
      </p>
      <div
        className="flex gap-2"
      >
        <div
          className="w-10 h-10 flex items-center justify-center p-2 rounded-full bg-purple-900"
        >
          <svg
            className="w-6 h-6 text-white fill-current  "
            viewBox="0 0 448 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
            />
          </svg>
        </div>
        <div
          className="w-10 h-10 flex items-center justify-center p-2 rounded-full bg-purple-900"
        >
          <svg
            className="w-6 h-6 text-white fill-current"
            viewBox="0 0 24 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M24 12.073c0-6.627-5.373-12-12-12s-12 5.373-12 12c0 5.99 4.388 10.954 10.125 11.854v-8.385H7.078v-3.47h3.047V9.43c0-3.007 1.792-4.669 4.533-4.669 1.312 0 2.686.235 2.686.235v2.953H15.83c-1.491 0-1.956.925-1.956 1.874v2.25h3.328l-.532 3.47h-2.796v8.385C19.612 23.027 24 18.062 24 12.073z"
            />
          </svg>
        </div>
        <div
          className="w-10 h-10 flex items-center justify-center p-1 rounded-full bg-purple-900"
        >
          <img
            alt="github"
            className=""
            src="github.svg"
          />
        </div>
      </div>
    </div>
    <div
      className="flex flex-col w-80 px-5 gap-1 sm:w-80 sm:px-5 "
    >
      <h1
        className="text-white text-xl mb-4"
      >
        MENU
      </h1>
      <a
        className="text-white font-semibold hover:text-purple-900"
        href="/"
        onClick={[Function]}
      >
        Home
      </a>
      <a
        className="text-white font-semibold hover:text-purple-900"
        href="/about"
        onClick={[Function]}
      >
        About
      </a>
      <a
        className="text-white font-semibold hover:text-purple-900"
        href="/contact"
        onClick={[Function]}
      >
        Contact
      </a>
    </div>
    <div
      className="flex flex-col w-80 px-5 gap-1 justify-start sm:w-80 sm:px-5"
    >
      <h1
        className="text-white text-xl mb-4"
      >
        SUPPORT
      </h1>
      <a
        className="text-white font-semibold hover:text-purple-900"
        href="/support"
        onClick={[Function]}
      >
        Support Center
      </a>
      <a
        className="text-white font-semibold hover:text-purple-900"
        href="/help"
        onClick={[Function]}
      >
        Help Desk
      </a>
      <a
        className="text-white font-semibold hover:text-purple-900"
        href="/faqs"
        onClick={[Function]}
      >
        FAQs
      </a>
    </div>
  </div>
</div>
`;
